pkg/sbom/sbom.go

```diff
@@ -78,6 +108,8 @@ func Generate(ctx context.Context, inputFilePath string, f io.Reader, distroID s
 	packageCollection := createdSBOM.Artifacts.Packages
 	packageCollection.Add(*apkPackage)
 
+	logger.Debug("finished Syft SBOM generation", "packageCount", packageCollection.PackageCount())
```

Comment:
I'd like to raise the level of this log entry for the moment to `Info` level.

Reply:
If we could add the package name too.

Reply:
Good idea, will do!